### PR TITLE
P0 - alert avatar badge components added

### PR DESCRIPTION
**Pull request by mayuresh-tech**
Work done:
Alerts, Avatar, and Badge added (Responsive)

To do:
Start working on the next P0 components

Bugs:
Sidebar:
If the user tests responsive then the sidebar elements get invisible.
aside needs to be given static height (for now set to 500vh)

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  border-radius: 50%;
+}
+
+.avatar-image-sm {
+  max-inline-size: 100%;
+  block-size: auto;
+  aspect-ratio: 1 / 1;
+  object-fit: cover;
+  height: 80px;
+  width: 80px;
```
Comment (kushalsourav) [inline components/avatar/avatar.css:33]: use rems insted of px

**Review by kushalsourav — APPROVED** [review]

File: components/badge/badge.css [hunk 2]
```diff
@@ -2,4 +2,90 @@
...
+
+.avatar {
+  position: relative;
+}
+
+.status-circle-lg {
+  width: 24px;
+  height: 24px;
+  border-radius: 50%;
+  border: 2px solid white;
```
Comment (kushalsourav) [inline components/badge/badge.css:15]: use variable for colors

**Review by kushalsourav — APPROVED** [review]

**Review by Bibin-Antony — APPROVED** [review]
kindly add a preview screenshot it will be easier to review.

File: components/avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  height: 120px;
+  width: 120px;
+  font-size: 2.5rem;
+}
+
+.avatar-text-sm {
+  background-color: var(--text-badge-color);
+  color: var(--light-text);
+  height: 80px;
+  width: 80px;
```
Comment (Ramandeep9898) [inline components/avatar/avatar.css:80]: u can use rem

**Review by Ramandeep9898 — COMMENTED** [review]
ui looks so gud and clean..
use rem....

File: components/avatar/avatar.css (hunk 3 above)
Comment (mayuresh-tech) [reply components/avatar/avatar.css:80]: Fix added

**Pull request merged**